great_expectations/experimental/datasources/postgres_datasource.py

```diff
@@ -198,7 +198,9 @@ def batch_request_options_template(
         return {p: None for p in self.column_splitter.param_names}
 
     def add_sorters(self, sorters: BatchSortersDefinition) -> TableAsset:
-        self._order_by = _batch_sorter_from_list(sorters)
+        # NOTE: (kilo59) we could use pydantic `validate_assignment` for this
+        # https://docs.pydantic.dev/usage/model_config/#options
+        self.order_by = _batch_sorter_from_list(sorters)
```

Comment:
Is this captured in a ticket anywhere?

Reply:
I don't think this is ticket worthy. We may not even want to do it.